### PR TITLE
Add org.gtk.Gtk3theme.Matcha-dark-azul

### PR DESCRIPTION
**Pull request by Paradoxeuh**

### CODE REVIEW

File: org.gtk.Gtk3theme.Matcha-dark-azul.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p ${FLATPAK_DEST}/share/appdata",
+                "cp org.gtk.Gtk3theme.Matcha-dark-azul.appdata.xml ${FLATPAK_DEST}/share/appdata",
+                "appstream-compose --basename=org.gtk.Gtk3theme.Matcha-dark-azul --prefix=${FLATPAK_DEST} --origin=flatpak org.gtk.Gtk3theme.Matcha-dark-azul"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "org.gtk.Gtk3theme.Matcha-dark-azul.appdata.xml"
```
Comment (LeandroStanger) [inline org.gtk.Gtk3theme.Matcha-dark-azul.json:40]: New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

File: README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Matcha-dark-azul on Flathub
```
Comment (barthalion) [inline README.md:1]: This is not really needed. Setup instructions are already there on our website and `flatpak update` will install matching theme automatically.

File: org.gtk.Gtk3theme.Matcha-dark-azul.json (hunk 1 above)
Comment (Paradoxeuh) [reply org.gtk.Gtk3theme.Matcha-dark-azul.json:40]: I'm not sure what you are talking about. When I read the quickstart, the xml I have seems legit. And when I watch all other themes on flathub, they all have similar appdata. Am I missing something?

**Pull request closed without merging**